cli-output/src/display.rs

```diff
@@ -125,6 +125,37 @@ pub fn println_signers(
     println!();
 }
 
+fn format_account_mode(message: &Message, index: usize, fee_payer_index: Option<usize>) -> String {
+    format!(
+        "{}r{}{}{}{}",
+        if message.is_signer(index) {
+            "S" // upper case because of signing is special by definition
+        } else {
+            "-"
+        },
+        if message.is_writable(index) {
+            "w" // comment for consistent rust fmt (no joking; lol)
+        } else {
+            "-"
+        },
+        if message.is_executable(index) {
```

Comment:
Perhaps a comment here to clarify that this represents an instruction program-id, but that there may be false negatives (ie. an account may be executable without being a designated program-id in the message).